test/index.mjs

```diff
@@ -0,0 +1,119 @@
+import { spawnSync } from 'child_process';
+import path from 'path';
+import { fileURLToPath } from 'url';
```

Comment:
This test is virtually the same except it runs run-binode*-esm in the package.json which in turn uses the binode-esm binary